pkg/planner/core/casetest/join/join_test.go

```diff
@@ -161,3 +161,88 @@ func TestJoinSimplifyCondition(t *testing.T) {
 				"  └─TableRowIDScan(Probe) 12.49 cop[tikv] table:t2 keep order:false, stats:pseudo"))
 	})
 }
+
+func TestKeepingJoinKeys(t *testing.T) {
```

Comment:
Can we merge these two test cases? It can reduce some initialization for the test environment.